c++/src/kj/async-io.c++

```diff
@@ -2629,6 +2629,22 @@ Promise<void> AsyncCapabilityStream::sendFd(int fd) {
   return promise.attach(kj::mv(fds));
 }
 
+Maybe<int> AsyncIoStream::releaseFd(Own<AsyncIoStream>&& stream) {
+  KJ_IF_MAYBE(fd, stream->getFd()) {
```

Comment:
I think you intended to use `doReleaseFd()` here?

Why didn't the test catch this?